---
layout: default
title:  Welcome to PoreCamp
---

# {{ page.title }}

PoreCamp 2015
14th December - 18th December 2015

**Organisers / Chief Instructors**

  - Nick Loman, Institute of Microbiology and Infection, University of Birmingham
  - Matt Loose, DeepSeq, University of Nottingham
  - Mick Watson, Edinburgh Genomics, The Roslin Institute
  - Josh Quick, University of Birmingham

**Special Invited Guest Lecturers**

  - Jared Simpson, Ontario Institute for Cancer Research
  - John Tyson, University of British Columbia, Vancouver
  - Justin O'Grady, University of East Anglia

**Teaching Assistants**

  - Emily Richardson, University of Birmingham
  - Judith Risse, Edinburgh Genomics

## Sponsors

Oxford Nanopore Technologies
Edinburgh Genomics
MicrobesNG
CLIMB
DeepSeq
NERC-NBAF

## Admission Information

**Cost**: 550 UK Pounds per person (accommodation not included)

Bursaries are available: 10 bursaries for NERC-NBAF students
CLIMB bursaries (?)

## Accommodation options

**Venue Birmingham**
£50 per night - only 14 rooms on the Monday, 28 rooms on the other days

## Admission criteria

Places are **strictly limited**. Places will be allocated according
to the following strict criteria:

You must:
 - be a member of the MinION Access Programme
 - have basic “hands-on” experience of the MinION and/or of MinION data analysis
 - have a background in either bioinformatics or wet lab work
 - be able to attend ALL FIVE days of the course

In the event the course is over subscribed, the instructors will
allocate places according to the strength of the application form
and the applicant's relevant experience.

## How to Apply

**Applications close on Friday, 6th November 2015 at 6pm UK time.**

Return this form to info@microbesng.uk with the subject line:

PORECAMP APPLICATION

	Name:
	Academic qualifications:
	Institution:
	Group (if applicable):
	Research interests (up to 100 words):
	Reason for wanting to attend course (up to 200 words):
	Experience with nanopore sequencing so far (up to 200 words):
	Background (check all that apply)
	[ ] Wet lab molecular biology
	[ ] Bioinformatics analysis
	What kind of samples do you intend to run on the MinION?
	If permitted, would you like to bring samples along to run? **

	** All data generated during the course will be made openly
	available online. Therefore do not propose to bring any
	samples that you would not like to release data for.

	If you would like to bring samples, please detail
	1) type of sample
	2) details of available DNA (volume, conc, quality)
	3) any potential biosafety implications, or specify none.


## Draft schedule

Day 1 -- Group Lectures 
History of second/third generation sequencing, nanopore in context
Introduction to nanopore sequencing
Introduction to applications
Understanding nanopore data
Nanopore workflow
Group session: Designing a nanopore experiment

Then rotating days of:
- Library preparation (10 people) - run by Josh
- Sequencing on the MinION (10 people) - run by Matt/John T - including real time streaming?
- Nanopore bioinformatics - run by Nick/Mick

Friday AM -- round-up, discussion

Curriculum

Library prep:
Sample input considerations
Fresh, frozen, FFPE
Protocols
Native, PCR, PreCR, Low-input
Run through SQK-MAP-006 on different samples (as above)
Josh run through and then students attempt their own

Sequencing:
Configuring laptop correctly
MinKNOW
Loading the instrument
Running MinKNOW
Interpreting platform QC
Interrogating runs in progress
Discussion of read until - perhaps live version if I can. Could try for balancing or enrichment of region in a small genome (suggestion lambda :-)
Metrichor
Data management
Interrogation of ongoing runs: minoTour, poretools

Bioinformatics
(copied from Edinburgh, please edit!)
Initial data handling:
Introduction to the Oxford Nanopore MinION sequencing technology
Organisation of raw reads (poRe/Poretools)
Extraction of FASTQ/A data (poRe/Poretools/minoTour)
Gathering and visualization of run statistics (poRe/Poretools/minoTour!)
Error correction and quality control (Nanocorr, NanoOK)
Alignment of reads (Last, BWA)
Event alignment (Nanopolish) - variant calling
Scaffolding of short reads (SSPACE-LongRead)
Hybrid de novo assembly (SPAdes)




