---
layout: default
title:  Timetable for Porecamp 2016
---

# {{ page.title }}

All group activities (coffee, lectures and reportage) and bioinformatics practicals will be held in the Exchange, Penryn Campus (Cornwall), University of Exeter.

# Monday 15th August: Introduction

- 12:30 : Registration and welcome buffet in Yellow Seminar Room
- 14:30 : Introduction to Porecamp (**Konrad Paszkiewicz**)
- 15:00 : Overview of the MinION platform (**Josh Quick**)
- 15:30 : Making a good library (**Justin Oâ€™Grady**)
- 16:30 : Designing your own experiment (Led by **Josh Quick**, but primarily evaluating QC of participant submitted samples, 1-2 minute talks by participants outlining their QC and their sample. Assigning individuals to rotation groups. Basic laboratory safety.)
- 18:30 : Settle into accommodation (collect keys from Glasney Lodge)
- 19:30 : Group dinner on campus (Lower Stannary room in the Daphne Du Maurier building)
 
# Tuesday 16th August: Practicals

Rotations: Group A Library Prep, Group B MinION Running, Group C Bioinformatics

- 09:00 : Group update in Yellow Seminar Room
- 09:15 : Practicals
- 13:00 : Lunchtime seminar: Understanding event calling and MinION data (**Matt Loose**)
- 14:00 : Practicals (continued)
 
# Wednesday 17th August: Practicals

Rotations: Group A MinION running, Group B Bioinformatics, Group C library Prep

- 09:00 : Group update in Yellow Seminar Room
- 09:15 : Practicals
- 13:00 : Lunchtime seminar: Title TBA (**John Tyson**)
- 14:00 : Practicals (continued)
 
# Thursday 18th August: Practicals

Rotations: Group A Bioinformatics, Group B library prep, Group C MinION running

- 09:00 : Group update in Yellow Seminar Room
- 09:15 : Practicals
- 13:00 : Lunchtime seminar: [Utilising nanopore sequencing for biothreat detection](lectures/PoreCamp2016_Lonsdale_BiothreatDetection.pptx) (**Claire Lonsdale**)
- 14:00 : Practicals (continued)

# Friday 19th August: Data analysis and conclusions

- 09:00 : Group update in Yellow Seminar Room 
- 09:15 : Seminar: Deploying a portable nanopore sequencing system (**Josh Quick**)
- 10:15 : Lab practical: Bioinformatics continued
- 13:00 : Lunch and wrap up (Nick Loman & Konrad Paszkiewicz)

# Teaching materials

Shared results

- [PoreCamp2016-Participant Google Drive folder](https://drive.google.com/open?id=0B3IdS6JcXsEWY3BwcUJzZ3lzR2M)

Library Prep

- XXXX

MinION Running

- XXXX

Bioinformatics tutorials

- [Logging onto CLIMB server desktop with X2GO](tutorials/PoreCamp2016-CLIMBDesktopWithX2Go.pdf)
- [Fungal strains](tutorials/PoreCamp2016_fungal_strains.docx)

- [00. Using the Linux command-line](tutorials/PoreCamp2016-00-Linux.pdf) (hand-out)
- [01. Using CLIMB](tutorials/PoreCamp2016-01-CLIMB.pdf) (Nick Loman)
- [02. Understanding your MinION data](tutorials/PoreCamp2016-02-MinIONData.pdf) (Nick Loman)
- [03. Mapping and variant calling on Ebola](../2015/tutorials/Mapping Practical.ipynb) (Nick Loman)
- De novo and hybrid assembly (Camilla Ip)

Analysis Notes

- BeachSeq (Nick Loman)
- Analysis tips (Nick Loman)
